common/engine/keyboardprocessor/tests/unit/kmx/meson.build

```diff
@@ -48,24 +48,29 @@ kmns = [
 #  '033 - caps alwyas off' # TODO: support capsononly, capsalwaysoff, shiftfreescaps
 ]
 
-prog_kmcomp = find_program('kmcomp')
+prog_kmcomp = find_program('kmcomp', required: false)
 
-# todo: if kmcomp is not found, use the .kmx in the source folder
-# therefore make kmcomp compile to the source folder and add to repo?
+if prog_kmcomp.found()
+  foreach kmn : kmns 
+    kmn_test = custom_target(
```

Comment:
This works  for me if I make it a configure_file, and copy the kmn as well
```
    kmn_test = configure_file(
      output: kmn + '.kmx',
      input: kmn + '.kmn',
      command : [prog_kmcomp, '-s', '@INPUT@']
    )
    kmn_source = configure_file(
      input: kmn + '.kmn',
      output: kmn + '.kmn',
      command : ['cp', '@INPUT@', '@OUTPUT@']
    )
```

The down side is that ninja keeps configuring for no obvious reason which can take a long time for compiling all the kmn multiple times.

Reply:
Tim pointed out during the ibus-keyman review that using `cp` here isn't cross-platform so we'll need to think of another way to do this

Reply:
See my comment above ... is there any reason we can't update meson?